### ISSUE TITLE
` character counted as space before closing parens?

### ISSUE THREAD

**Issue by paulcpederson** [feross/standard#34]
Using the new ES6 syntax, I have several template strings passed as arguments to a function or method, formatted like this:

```
dothing(`url${variable}`)
```

Standard reports an error at the end of the line. The error text is:

```
Illegal space before closing round bracket
```

**Comment by feross** [comment]
Confirmed.

**Comment by feross** [comment]
Thanks for reporting! Fixed in 2.6.2.

**Comment by paulcpederson** [comment]
:+1:
